### PR TITLE
len(catalog) sometimes reported as 1

### LINKED ISSUE #70: BUG: double click pager ">" button resets page offset to 0

### PR DESCRIPTION
**Pull request by prjemian**
- close #70

### CODE REVIEW

File: gemviz23/demo/resultwindow.py [hunk 1]
```diff
@@ -78,34 +79,40 @@ def headerData(self, section, orientation, role=QtCore.Qt.DisplayRole):
 
     def doPager(self, action, value = None):
         print(f"doPager {action =}, {value =}")
+
+        catalog_length = self.catalog_length()
+        offset = self.pageOffset()
+        size = self.pageSize()
+        print(f"{catalog_length=} {offset=}  {size=}")
```
Comment (olliviergassant) [inline gemviz23/demo/resultwindow.py:86]: We can remove this print statement

**Review by olliviergassant — APPROVED** [review]
- LGTM
- We can remove the print statement at line 86

File: gemviz23/demo/resultwindow.py (hunk 1 above)
Comment (prjemian) [reply gemviz23/demo/resultwindow.py:86]: Taking out all similar prints in this module by comment

**Review by rodolakis — COMMENTED** [review]
done

**Pull request merged**